recipe/meta.yaml

```
@@ -29,8 +33,8 @@ source:
     
 
 build:
-  number: 3
-  skip: True  # [win]
+  number: 4
+  skip: True  # [win or name_suffix=="3.6"]
```

Do I dare try to build for windows? It should be supported...
```suggestion
  number: 4
  skip: True  # [name_suffix=="3.6"]
```

---

I think it's best to leave that to a new PR, but I'm definitely interested.

---

I removed `win` from the skip but the CI did not start :(. What did I miss?